code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -248,6 +248,37 @@
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 	new /obj/item/weapon/grenade/chem_grenade/cleaner(src)
 
+/obj/item/weapon/storage/belt/lazarus
+	name = "trainer's belt"
+	desc = "For the mining master, holds your lazarus capsules."
+	icon_state = "lazarusbelt"
+	item_state = "lazbelt"
+	w_class = 4
+	max_w_class = 1
+	max_combined_w_class = 6
+	storage_slots = 6
+	can_hold = list("/obj/item/device/mobcapsule")

```

Comment:
I'd consider letting the belt also hold the injectors. Then you can carry the capsules and injectors together, rather than dividing them across multiple storage containers.